generators/app/templates/src/app/components/CustomTextInput/components/ShowPassword/index.js

```diff
@@ -14,8 +14,8 @@ const ShowPassword = ({ onShowPassword, passwordVisible }) => (
 );
 
 ShowPassword.propTypes = {
-  onShowPassword: PropTypes.func.isRequired,
-  passwordVisible: PropTypes.bool.isRequired
+  passwordVisible: PropTypes.bool.isRequired,
+  onShowPassword: PropTypes.func.isRequired
```

Comment:
mmm i think it was okay before

Reply:
it occurs beacuse the react/sort-prop-types puts callbacksLast in true. I think that it is not bad.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md